Dockerfile

```diff
@@ -1,9 +1,27 @@
 FROM ubuntu:16.04
-RUN apt-get update && apt-get install -y groovy2 python-pip build-essential python-dev libssl-dev curl libffi-dev sudo git-core libxml2-utils
+RUN apt-get update \
+ && apt-get install -y \
+      groovy2 \
+      python-pip \
+      build-essential \
+      python-dev \
+      libssl-dev \
+      curl \
+      libffi-dev \
+      sudo \
+      git-core \
+      libxml2-utils \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
Perhaps using ``apt-get clean`` would also be useful here if we're intending to do some cleanup to maintain a clean image?

Reply:
According to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/ this should already be done.